plugins/engines/ospray/OSPRayEngine.cpp

```diff
@@ -46,16 +48,17 @@ OSPRayEngine::OSPRayEngine(int argc, const char** argv,
                     << std::endl;
     }
 
+    RenderingParameters& rp = _parametersManager.getRenderingParameters();
+    if (!rp.getModule().empty())
+        ospLoadModule(rp.getModule().c_str());
```

Comment:
Maybe ospLoadModule returns a value that would be a better check for knowing if the module is loaded?